### PR TITLE
feat: use contractStarter to start, for example, postalSvc

### PR DESCRIPTION
**Pull request by dckc**
Probably a long way to go, but it does pass 1 test.

resume work from ...
 - https://github.com/Agoric/agoric-sdk/pull/8640

### CODE REVIEW

**Comment by dckc** [comment]
@0xpatrickdev I think this is about ready. Any thoughts before I merge it?

File: contract/test/test-start1.js
Comment (0xpatrickdev) [inline contract/test/test-start1.js:1]: Any reason not to call this `test-contractStarter.js`?

**Review by 0xpatrickdev — APPROVED** [review]
Light review, but it's a prototype and I have confidence after you walked me through the tests earlier today.

File: contract/test/test-start1.js
Comment (dckc) [reply contract/test/test-start1.js:1]: I'm not sure... I was thinking I might test starting other things... or something.

**Comment by dckc** [comment]
recording a bit more history from agoric-sdk before I close https://github.com/Agoric/agoric-sdk/pull/8640

2023-12-09 13:40 e9c2b74d1 test: start 1 contract with contractStarter
2023-11-21 00:32 437af8802 feat: contractStarter (WIP)
2023-12-09 13:39 484128f46 test: factor boot-tools out of test-postalSvc
2023-12-10 16:26 9265fe84c refactor(start-postalSvc): separate install / start functions
2023-12-09 12:11 9c75409c8 refactor(start-postalSvc): one function that can be stringified

**Pull request merged**